### DISCUSSION TITLE
Seattle City Parks

### DISCUSSION THREAD

**Discussion by lucretiachen99** [jakobzhao/geog495#74] [Final Project]
Application: https://lucretiachen99.github.io/seattlecityparks/index.html
Github: https://github.com/lucretiachen99/seattlecityparks
Description: The goal of this project is to locate the remaining city parks in the Seattle area to give us a better idea of how much area is dedicated to parks compared to how much is built into other things such as roads, houses, malls, etc. This map displays a list of parks and flys to the location the user wants to view. It highlights all area of the city parks on the map.
![image](https://user-images.githubusercontent.com/91760537/145734332-5c1a1dbe-bcb3-4d6a-b97c-26bc050d2fe3.png)

**Comment by Anemnox** [comment]
Hey! The start of your map looks really cool! I like the color scheme of the basemap and how you've matched the side menu with the same schema. I think once you've added the parks data, the overall map will come out pretty well.

**Comment by FaisalTF** [comment]
I liked your decisions with the base map, a natural look will have parks pop off in green and match the narrative! Trying to constructively think about where this could improve, besides the data points not plugging in, I think the some contrast to the side bar's aesthetics could help distinguish the two sections and navigating them, as it somewhat blends in with its current theme.

**Comment by RensenJiang** [comment]
Hi, I think it would be a good map if you can choose a different color scheme, also the information bar on the left are all undefined, you might want to double-check your data loading process.
